packages/nextjs/hooks/scaffold-eth/useNativeCurrencyPrice.ts

```diff
@@ -8,28 +8,28 @@ const enablePolling = false;
 
 /**
  * Get the price of ETH based on ETH/DAI trading pair from Uniswap SDK
```

Comment:
Maybe `Get the price of Native currency based on NativeToken/DAI...` please feel free to improve it 🙌

Reply:
Yup should have updated this thanks